src/spdx/document_utils.py

```diff
@@ -9,12 +9,23 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 from typing import List
-
+from typing import Union
 from spdx.model.document import Document
+from spdx.model.snippet import Snippet
+from spdx.model.package import Package
+from spdx.model.file import File
 
 
 def get_contained_spdx_element_ids(document: Document) -> List[str]:
     element_ids = [file.spdx_id for file in document.files]
     element_ids.extend([package.spdx_id for package in document.packages])
     element_ids.extend([snippet.spdx_id for snippet in document.snippets])
     return element_ids
+def get_element_from_spdx_id(document: Document, spdx_id: str) -> Union[Package, File, Snippet, None]:
```

Comment:
PEP8 requires two blank lines between methods.
Which IDE are you using for your python development? There might be a plugin for your IDE to detect PEP8 violations.

Reply:
I am using VSCode